ext/opentelemetry-ext-requests/tests/test_requests_integration.py

```diff
@@ -183,6 +185,45 @@ def test_distributed_context(self):
         finally:
             propagators.set_global_httptextformat(previous_propagator)
 
+    def test_apply_custom_attributes_on_span(self):
+        resource = resources.Resource.create({})
+        result = self.create_tracer_provider(resource=resource)
```

Comment:
You can avoid creating a new one and can reuse `self.tracer_provider`, again, it was only needed for the `test_custom_tracer_provider`.

Reply:
Done.